src/app/core/default-values/x-dynamic-placeholder/dynamic-placeholder-value.service.ts

```diff
@@ -70,4 +54,27 @@ export class DynamicPlaceholderValueService extends DefaultValueStrategy {
       targetFormControl.setValue(value);
     }
   }
+
+  /**
+   * Get the current value for a given placeholder.
+   * @param placeholder The placeholder string to resolve.
+   * @returns The value for the given placeholder or `undefined` if not found.
+   */
+  getPlaceholderValue(placeholder: string): string | Date | undefined {
+    switch (placeholder) {
+      case PLACEHOLDERS.CURRENT_USER:
+        let userId = this.currentUser.value?.getId();
+        return userId;
+      case PLACEHOLDERS.NOW:
+        let date = new Date();
+        return date;
+      default:
+        Logging.warn(
```

Comment:
```suggestion
        Logging.debug(
```

turns out this happens quite a lot now. So let's tone this down to "debug" log level only (otherwise it'll show up on our central monitoring platform)